Items/07_Items.txt

```diff
@@ -43,6 +43,12 @@ from multiple angles to capture the whole object. Up to 20m^3 of space.
 Wearables
 ==============================
 
+Dog Collar ($45):
+    This 30cm wide collar comes with up to 18 characters of monogram and a micro
```

Comment:
The width of a collar usually refers to the distance across the collar, not the length around; 30cm is really wide compared to the length of most dogs' necks.
In addition, current husbandry standards greatly favor harnesses over collars; harnesses don't choke a dog or cause spinal injuries when pressure is put on them.